seqr/views/apis/data_manager_api_tests.py

```diff
@@ -1031,13 +1032,21 @@ def _test_basic_data_loading(data, num_parsed_samples, num_loaded_samples, new_s
         mock_send_slack.assert_has_calls([
             mock.call(
                 'seqr-data-loading',
-                f'0 new RNA {params["message_data_type"]} samples are loaded in <https://test-seqr.org/project/R0001_1kg/project_page|1kg project nåme with uniçøde>\n``````',
+                f'0 new RNA {params["message_data_type"]} samples are loaded in <a href=https://test-seqr.org/project/R0001_1kg/project_page>1kg project nåme with uniçøde</a>\n``````',
```

Comment:
why did the slack notification change here? Slack does not render html, it renders its own markdown - you can see a successful example of this message here: https://the-tgg.slack.com/archives/CA620K989/p1689367904465639

Reply:
this was a mistake, fixed it